packages/livechat/src/lib/hooks.ts

```diff
@@ -247,6 +247,27 @@ const api = {
 		});
 	},
 
+	transferChat: async (department: string) => {
+		const {
+			config: { departments = [] },
+			room,
+		} = store.state;
+
+		const dep = departments.find((dep) => dep._id === department || dep.name === department)?._id || '';
+
+		if (!dep) {
+			throw new Error(
+				'The selected department is invalid. Check departments configuration to ensure the department exists, is enabled and has at least 1 agent',
```

Comment:
If this is an error customers should see, I believe we can improve the writing to make it more generic, since for visitors none of the info here is actionable.

And then we print to the console the full error. wdyt?

Reply:
This error does not show up in the widget, it throws in the console only